terraform/aptos-node-testnet/variables.tf

```diff
@@ -164,12 +164,12 @@ variable "validator_instance_max_num" {
 
 variable "utility_instance_type" {
   description = "Instance type used for utilities"
-  default     = "t3.medium"
+  default     = "t3.2xlarge"
```

Comment:
have you checked if t3 is still the best instance type to use in this case?

Reply:
General purpose cost-per-performance, yea it seems so.